README.md

```diff
@@ -206,8 +206,13 @@ used by default (with no profile specified).
 To start the development environment, run:
 
 ```bash
+# Make sure to export your UID, or else Spark may run into permission problems
+# when trying to write to files in your local directory
+export UID
+# Start the development services
```

Comment:
Folding in this unrelated docs change, per our IRL discussion. I added some more flavor to the comments to clarify their meaning, too.